src/features/messages/MessagesStore.ts

```diff
@@ -0,0 +1,153 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
Can we please have tests for this? An example can be found in https://github.com/pastelnetwork/pastel-electron-wallet/tree/a69d01a4d68460eda71bde7866fc81f9483127ae/src/features/pastelPrice/__tests__

Reply:
Got it.